scripts/setup-centos9.sh

```diff
@@ -31,18 +31,28 @@ set -efx -o pipefail
 SCRIPTDIR=$(dirname "${BASH_SOURCE[0]}")
 source $SCRIPTDIR/setup-helper-functions.sh
 NPROC=$(getconf _NPROCESSORS_ONLN)
-export CFLAGS=$(get_cxx_flags)  # Used by LZO.
-export CXXFLAGS=$CFLAGS  # Used by boost.
-export CPPFLAGS=$CFLAGS  # Used by LZO.
+export CXXFLAGS=$(get_cxx_flags) # Used by boost. 
+export CFLAGS=${CXXFLAGS//"-std=c++17"/} # Used by LZO.
 CMAKE_BUILD_TYPE="${BUILD_TYPE:-Release}"
 BUILD_DUCKDB="${BUILD_DUCKDB:-true}"
-export CC=/opt/rh/gcc-toolset-12/root/bin/gcc
-export CXX=/opt/rh/gcc-toolset-12/root/bin/g++
+USE_CLANG="${USE_CLANG:-false}"
 
 function dnf_install {
   dnf install -y -q --setopt=install_weak_deps=False "$@"
 }
 
+function install_and_setup_clang15 {
+  dnf_install clang15 gcc-toolset-13-libatomic-devel
+  CLANG_NUM_ALTERNATIVES=`alternatives --list | grep "clang " | wc -l`
```

Comment:
nit: I am a bit ignorant about this, Can you add a brief comment ?

Reply:
Will do.